control-plane/connect-inject/peering_acceptor_controller.go

```diff
@@ -251,9 +269,6 @@ func (r *PeeringAcceptorController) createK8sPeeringTokenSecretWithOwner(ctx con
 	secretName := acceptor.Secret().Name
 	secretNamespace := acceptor.Namespace
 	secret := createSecret(secretName, secretNamespace, acceptor.Secret().Key, resp.PeeringToken)
-	if err := controllerutil.SetControllerReference(acceptor, secret, r.Scheme); err != nil {
```

Comment:
Removed this as it causes the secret to be deleted as soon as it is migrated between cluster. We will add watchers for secrets in a subsequent PR.